app/models/claim/data_report_request.rb

```diff
@@ -21,7 +23,9 @@ def to_csv
           csv << [
             claim.reference,
             claim.teacher_reference_number,
-            claim.full_name
+            claim.full_name,
+            claim.email_address,
+            claim.date_of_birth
```

Comment:
A heads up from past experience which I'm not certain if it applies here or not - you may want to double check that Excel will happily understand this date without ambiguity (eg "04/08/2020" will upset it as it won't know if we're talking April or August). You may want to be explicit about the output format.

Reply:
Checked in excel and it worked fine.